docs/index.html

```diff
@@ -16,70 +16,22 @@ <h1>Syntax Tree</h1>
         <span><button type="button" id="format" disabled>Format</button></span>
 
         <div class="toggles">
-          <span><button type="button" value="prettyPrint" disabled>AST</button></span>
-          <span><button type="button" value="disasm" disabled>ISEQ</button></span>
+          <select>
```

Comment:
Let's start the `select` as disabled until the imports resolve. Also we should add an aria-label here for accessibility